artifacts

```diff
@@ -16,7 +16,7 @@ set_output() {
 
 
 setup() {
-    NAME="github:${REPOSITORY}:staging-${COMMIT_TIMESTAMP}.${COMMIT_SHORT_REVISION}.${WORKFLOW_NAME}.${RUN_NUMBER}"
+    NAME="github:${REPOSITORY}:staging-${COMMIT_TIMESTAMP}.${COMMIT_SHORT_REVISION}.${WORKFLOW_NAME}.${RUN_NUMBER}${RUN_ATTEMPT}"
```

Comment:
As I explained yesterday, I'd like to have RUN_ATTEMPT separated from RUN_NUMBER by a character, otherwise this is not consistent or unique.